src/app/components/icons/IconDrawer.tsx

```diff
@@ -150,16 +172,82 @@ export const IconDrawer: React.FC = () => {
+                                            id="icon-size-select"
+                                            value={iconSize}
+                                            onChange={handleChange}
+                                        >
+                                            <MenuItem value={18}>18dp</MenuItem>
+                                            <MenuItem value={24}>24dp</MenuItem>
+                                            <MenuItem value={36}>36dp</MenuItem>
+                                            <MenuItem value={48}>48dp</MenuItem>
+                                        </Select>
+                                    ) : (
+                                        <Select
+                                            labelId="icon-size-select-label"
+                                            id="icon-size-select"
+                                            value={iconSize}
+                                            onChange={handleChange}
+                                        >
+                                            <MenuItem value={24}>24dp</MenuItem>
+                                            <MenuItem value={48}>48dp</MenuItem>
+                                        </Select>
+                                    )}
+                                </FormControl>
+                                <FormControl className={classes.formControl}>
+                                    <InputLabel id="icon-color-select-label">Select a Color:</InputLabel>
+                                    {selectedIcon.isMaterial ? (
+                                        <Select
+                                            labelId="icon-color-select-label"
+                                            id="icon-color-select"
+                                            value={iconColor}
+                                            onChange={handleColorChange}
+                                        >
+                                            <MenuItem value={'Black'}>Black</MenuItem>
+                                            <MenuItem value={'White'}>White</MenuItem>
+                                        </Select>
+                                    ) : (
+                                        <Select
+                                            labelId="icon-color-select-label"
+                                            id="icon-color-select"
+                                            value={iconColor}
+                                            onChange={handleColorChange}
+                                        >
+                                            <MenuItem value={'Black'}>Black</MenuItem>
+                                            <MenuItem value={'White'}>White</MenuItem>
+                                            <MenuItem value={'Blue'}>Blue</MenuItem>
+                                            <MenuItem value={'Gray'}>Gray</MenuItem>
```

Comment:
You could make the values here the explicit color that we want to use (so you won't need any logic to look them up in the download function):

```
<MenuItem value={Colors.black[500]}>Black</MenuItem>
```

Reply:
I just realized this would overcomplicate the logic in the PNGs, so we can probably stick with what we have (maybe just lowercase the `values` so we don't have to do it later).